manual/distance_tables.tex

```diff
@@ -0,0 +1,91 @@
+\section{Particles and distance tables}
+\label{sec:distance_tables}
+
+\subsection{ParticleSets}
+The \texttt{ParticleSet} class stores particle positions and attributes (charge, mass, etc).
+
+The \texttt{R} member stores positions.
+For calculations, the \texttt{R} variable needs to be transferred to the structure-of-arrays (SoA) storage in \texttt{RSoA}.   This is done by the \texttt{update} method.
+In the future the interface may change to use functions to set and retrieve positions
+so the SoA transformation of the particle data can happen automatically.
+
+A particular distance table is retrieved with \texttt{getDistTable}.
+The first table in the list is the table for like particles (electron-electron or ion-ion), and is always present.
```

Comment:
I think this behavior has been removed. The first table only depends on the first call of addTable.

Reply:
Yes, this has changed.   Removed that sentence.